network/vitalii/README-NAT.md

```diff
@@ -0,0 +1,35 @@
+Task: Know, understand what NAT is, why it is needed, what is the principle of operation, where it is used
+___
+
+# NAT (Network Address Translation)
+NAT - is a mechanism in TCP/IP networks that allows the IP address to be changed in the header of a packet passing through a traffic r
```

Comment:
typo: `\n` after r in the end of the line

Reply:
fixed [commit](https://github.com/0lmi/devops-training/pull/38/commits/2b9a7cd01836ea1011e0e3635a3be4e8d1d6efb8)